common/tasks/rosa-hcp/generate-cluster-name.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: generate-cluster-name
+spec:
+  results:
+    - name: cluster-name
+      description: Openshift cluster name
+  steps:
+    - name: generate-cluster-name
+      image: registry.access.redhat.com/ubi9/openssl:9.1-2
+      script: |
+        #!/usr/bin/env bash
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+        
+        CLUSTER_NAME="ci-$( openssl rand -hex 5 )"
```

Comment:
CLUSTER_NAME="ci-$( openssl rand -hex 5 )"

Reply:
@flacatus what do you mean? it's totally same as mine.

Reply:
fixed.